internal/db/schema/migrations/postgres/11/01_server_type_enum.up.sql

```diff
@@ -0,0 +1,17 @@
+begin;
+
+create table server_type_enm (
+  name text primary key not null,
```

Comment:
No need for `not null` since primary keys are always not null.

Reply:
Also, there should not be a comma at the end of this line since the constraint is a column constraint. Adding the comma makes the constraint a table constraint. Postgres will allow it but other database will not.

Reply:
Should both be addressed!